go.mod

```diff
@@ -70,3 +70,5 @@ require (
 	google.golang.org/grpc v1.63.2 // indirect
 	google.golang.org/protobuf v1.34.0 // indirect
 )
+
+replace github.com/vmware/go-vcloud-director/v2 => github.com/adambarreiro/go-vcloud-director/v2 v2.17.0-alpha.1.0.20240606095202-a2a846adaa90
```

Comment:
reminder to remove this replacement before merging

Reply:
Done